src/services/panel.ts

```diff
@@ -217,9 +228,30 @@ export function sortLevelTransformation() {
               return 0;
             }
             const aName: string | undefined = a.fields[1].config.displayNameFromDS;
-            const aVal = aName?.includes('error') ? 4 : aName?.includes('warn') ? 3 : aName?.includes('info') ? 2 : 1;
+            const aVal = aName?.match(new RegExp(CRITICAL_LEVEL_FIELD_NAME_REGEX))
```

Comment:
Same here, it's creating a regex from a regex. Unless I'm missing a good reason to do that.